Source/santactl/Commands/SNTCommandStatus.m

```diff
@@ -310,6 +310,9 @@ - (void)runWithArguments:(NSArray *)arguments {
       printf("  %-25s | %s\n", "Push Notifications",
              (pushNotifications ? "Connected" : "Disconnected"));
       printf("  %-25s | %s\n", "Bundle Scanning", (enableBundles ? "Yes" : "No"));
+    }
+
+    if (enableTransitiveRules) {
```

Comment:
I think we should move this up to `Daemon Info` above (like other things that can be configured via profile or sync server)